charts/redis/templates/Secret.yaml

```diff
@@ -1,3 +1,5 @@
+{{- if not .Values.sealed_secrets.enabled }}
+{{- if .Values.managed_redis.disabled }}
```

Comment:
We don't need this line as from our application we won't call this chart if we use managed_redis.

Reply:
It's removed